packages/bundler-plugin-core/src/index.ts

```diff
@@ -253,90 +253,15 @@ export function sentryCliBinaryExists(): boolean {
 }
 
 export function createRollupReleaseInjectionHooks(injectionCode: string) {
-  const virtualReleaseInjectionFileId = "\0sentry-release-injection-file";
-
   return {
-    resolveId(id: string) {
-      if (id === virtualReleaseInjectionFileId) {
-        return {
-          id: virtualReleaseInjectionFileId,
-          external: false,
-          moduleSideEffects: true,
-        };
-      } else {
-        return null;
-      }
-    },
-
-    load(id: string) {
-      if (id === virtualReleaseInjectionFileId) {
-        return injectionCode;
-      } else {
-        return null;
-      }
-    },
-
-    transform(code: string, id: string) {
-      if (id === virtualReleaseInjectionFileId) {
-        return null;
-      }
-
-      if (id.match(/\\node_modules\\|\/node_modules\//)) {
-        return null;
-      }
-
-      if (![".js", ".ts", ".jsx", ".tsx", ".mjs"].some((ending) => id.endsWith(ending))) {
-        return null;
-      }
-
-      const ms = new MagicString(code);
-
-      // Appending instead of prepending has less probability of mucking with user's source maps.
-      // Luckily import statements get hoisted to the top anyways.
-      ms.append(`\n\n;import "${virtualReleaseInjectionFileId}";`);
-
-      return {
-        code: ms.toString(),
-        map: ms.generateMap(),
-      };
-    },
+    banner: injectionCode,
```

Comment:
Hmm wasn't there some requirement for us to be below `use strict;` or something like that? But other than that, good catch 😅